content/spin/writing-apps.md

```diff
@@ -270,12 +270,38 @@ If your files list would match some files or directories that you _don't_ want i
 
 > By default, Spin takes a snapshot of your included files, and components access that snapshot. This ensures that when you test your application, you are checking it with the set of files it will be deployed with. However, it also means that your component does not pick up changes to the files while it is running. When testing a Web site, you might want to be able to edit a HTML or CSS file, and see the changes reflected without restarting Spin. You can tell Spin to give components access to the original, "live" files by passing the `--direct-mounts` flag to `spin up`.
 
+## Adding Environment Variables to Components
+
+Environment variables can be provided to components via the Spin application manifest.
+
+To do this, use the `environment` field in the component manifest:
+
+```toml
+[[component]]
+environment = { PET = "CAT", FOOD = "WATERMELON" }
+```
+
+The field accepts a map of environment variable key/value pairs. They are mapped inside the component at runtime.
+
+The environment variables can then be accessed inside the component. For example, in Rust:
+
+```rs
+#[http_component]
+fn handle_hello_rust(req: Request) -> Result<Response> {
+    let response format!("My {} likes to eat {}", std::env::var("PET")?, std::env::var("FOOD")?);
```

Comment:
I am very late to this party but I think this is missing an `=` sign?

Reply:
Nice catch Ivan! Heres a fix https://github.com/fermyon/developer/pull/529